assets/images/close.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I found that it is better to use @yscik's `<svg><use ...></svg>` approach, because it's content is not affected by the resets introduced with the course theme frame. However those svg icons are not rendered when course theme is not enabled. But for contact-teacher it needs to work with & without course theme enabled.